src/main/java/ca/corefacility/bioinformatics/irida/config/security/IridaOauthSecurityConfig.java

```diff
@@ -65,7 +66,7 @@ public ClientDetailsUserDetailsService clientDetailsUserDetailsService(
 	}
 
 	@Bean
-	public WebResponseExceptionTranslator exceptionTranslator() {
+	public WebResponseExceptionTranslator<OAuth2Exception> exceptionTranslator() {
 		return new CustomOAuth2ExceptionTranslator();
```

Comment:
Compiler was giving warning about `Raw Type`, so I changed to the specific type i.e. `WebResponseExceptionTranslator<OAuth2Exception>` that `new CustomOAuth2ExceptionTranslator()` returns.